### DISCUSSION TITLE
Any plans to add lyric option?

### DISCUSSION THREAD

**Discussion by Matticusnicholas** [intel/openvino-plugins-ai-audacity#76] [Ideas]
Hoping in the future there will be a lyrics feature in the music generation option.  Would finally have a replacement for suno.

**Comment by RyanMetcalfeInt8** [comment]
Hi @Matticusnicholas,

It's possible -- We'd have to find a nice open source model that does it well. I think the closest open source model that does something right now is suno's bark model (https://github.com/suno-ai/bark). But the models that power their Suno AI web apps are closed source.

Cheers!
